app/src/features/FhirResourceTree/FhirResourceTree.tsx

```diff
@@ -95,6 +99,28 @@ const FhirResourceTree = (): JSX.Element => {
     }
   }, [expandedNodes, hasAlreadyExpandedToTarget, nodeAncestorsIds]);
 
+  const createStaticInputWithFixedValue = (
+    node: ElementNode,
+    inputGroup: InputGroup
+  ) => {
+    const fixedValueKey = Object.keys(
+      node.definitionNode.definition
+    ).find((elementDefinitionKey) => elementDefinitionKey.includes("fixed"));
+    if (fixedValueKey && node) {
+      const fixedValue =
+        node.definitionNode.definition[
+          fixedValueKey as keyof IElementDefinition
+        ];
+      if (node.type && primitiveTypes.includes(node.type) && fixedValue)
+        createStaticInput({
```

Comment:
By doing all the changes, you can factorise those 2 if like so :

```suggestion
    if (fixedEntry && node.kind === "primitive") {
      const fixedValue = fixedEntry[1];
      await createStaticInput({
```